drupal/manifests/drupal-secrets.yaml

```diff
@@ -0,0 +1,11 @@
+#helm:generate helmc tpl -d tpl/values.toml -o manifests/drupal-secrets.yaml $HELM_GENERATE_FILE
+apiVersion: v1
+kind: Secret
+metadata:
+  name: drupal
```

Comment:
> drupal_secret, drupal_rc and drupal_svc

What's the value of doing that? I thought that `kubectl get` requires a [resource type] sub-command so there is no way to get mixed up output. 

In the newest helm example [1], the resource names are the same regardless of the type of resource.

So in my opinion, we should not add the suffix.

[1] https://github.com/kubernetes/helm/tree/master/docs/examples/nginx/templates